Gems/PhysX/Code/Source/Material/PhysXMaterial.cpp

```diff
@@ -273,6 +312,74 @@ namespace PhysX
         m_density = AZ::GetClamp(density, MaterialConstants::MinDensityLimit, MaterialConstants::MaxDensityLimit);
     }
 
+    bool Material::IsCompliantContactModeEnabled() const
+    {
+#if (PX_PHYSICS_VERSION_MAJOR >= 5)
+        return (m_pxMaterial->getFlags() & physx::PxMaterialFlag::eCOMPLIANT_CONTACT) != 0;
+#else
+        return false;
+#endif
+    }
+
+    void Material::EnableCompliantContactMode([[maybe_unused]] bool enabled)
+    {
+#if (PX_PHYSICS_VERSION_MAJOR >= 5)
+        m_pxMaterial->setFlag(physx::PxMaterialFlag::eCOMPLIANT_CONTACT, enabled);
```

Comment:
```suggestion
        return m_pxMaterial->getFlags().isSet(physx::PxMaterialFlag::eCOMPLIANT_CONTACT);
```

Reply:
Clang couldn't compile since PxFlag type provided some ambiguity here
```
'PxFlags<physx::PxMaterialFlag::Enum, unsigned short>' and 'physx::PxU16' (aka 'unsigned short'))
```
I proposed suggestion.

Reply:
`return m_pxMaterial->getFlags().isSet(physx::PxMaterialFlag::eCOMPLIANT_CONTACT);`

That's much better, thanks! I'll change it.